apps/codeforafrica/src/lib/api.ghost/posts.js

```diff
@@ -0,0 +1,97 @@
+import fs from "fs";
+import path from "path";
+
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+const cacheDir = path.join(process.env.PWD, "public/data");
+
+function ghostAPI() {
+  return new GhostContentAPI({
+    url: process.env.GHOST_URL,
+    key: process.env.GHOST_API_KEY,
+    version: process.env.GHOST_API_VERSION || "v5.0",
+  });
+}
+
+function transformPost(post) {
+  const {
+    customExcerpt,
+    slug,
+    primaryTag,
+    publishedAt: publishedAtRaw,
+    excerpt: originalExcerpt,
+    ...other
+  } = convertToCamelCase(post);
+
+  const excerpt = customExcerpt || originalExcerpt || null;
+  const href = `/${primaryTag.slug}/${slug}`;
+  const publishedAt = new Date(publishedAtRaw).toLocaleDateString("en", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  });
+  return {
+    ...other,
+    excerpt,
+    publishedAt,
+    href,
+    slug,
+    primaryTag,
+  };
```

Comment:
```suggestion
  const excerpt = customExcerpt || originalExcerpt || null;
  const href = `/${primaryTag.slug}/${slug}`;
  const publishedAt = new Date(publishedAtRaw).toLocaleDateString("en", {
    year: "numeric",
    month: "short",
    day: "numeric",
  });
  const tags = originalTags
    .filter((t) => equalsIgnoreCase(t.name, "opportunities") || equalsIgnoreCase(t.name, "stories"))
    .map((t) => t.name);
  return {
    ...other,
    excerpt,
    href,
    primaryTag,
    publishedAt,
    slug,
    tags,
  };
```
Opportunites and Stories should only be in primary tag.

Reply:
@kilemensi Doesn't using this remove all the other tags, and only remain with the primary tag? Should we instead use  `!equalsIgnoreCase` so as to include other tags other than the primary tag?

Reply:
💯 @kelvinkipruto . It should be
```js
    .filter((t) => !(equalsIgnoreCase(t.name, "opportunities") || equalsIgnoreCase(t.name, "stories")))
```